packages/button/src/MenuButton.js

```diff
@@ -19,7 +19,6 @@ const toggleButtonStyles = {
 }
 
 const MenuButtonContainer = styled.div`
- display: inline-flex;
  justify-content: center;
```

Comment:
this seems specific to flex to. shouldn't we remove that as well?

Reply:
Yeah, will do.